README.md

```diff
@@ -297,9 +297,6 @@ sudo apt install -y libapache2-mod-php8.2 \
   php8.2-xml \
   php8.2-zip
 
-# fix imagick
-sudo apt install -y php-imagick imagick
```

Comment:
A bit suspicious that installing `imagick` itself isn't in the "fix imagick" anymore.  
Perhaps it's free as a `php-imagick` dependency but wanted to double check this before I merge

Reply:
I moved the package installation down to line 315.

Reply:
The latest commit explicitly installs it. Adding it is faster than asking if dependency resolution will handle it.